stock-managment/src/main/java/org/dhis2/android/rtsm/ui/managestock/ManageStockViewModel.kt

```diff
@@ -79,27 +78,37 @@ class ManageStockViewModel @Inject constructor(
     private val _hasData = MutableStateFlow(false)
     val hasData = _hasData
 
-    private val _networkState = MutableLiveData<OperationState<LiveData<PagedList<StockItem>>>>()
-    val operationState: LiveData<OperationState<LiveData<PagedList<StockItem>>>>
-        get() = _networkState
-
-    private val _screenState: MutableStateFlow<TableScreenState> = MutableStateFlow(
+    private val _screenState: MutableLiveData<TableScreenState> = MutableLiveData(
         TableScreenState(emptyList(), false)
     )
-    val screenState: StateFlow<TableScreenState> = _screenState
```

Comment:
Why did you use, Livedata instead of StateFlow.  I just want to understand your choice, because I think also we can set values using StateFlow.  I am not saying that it is wrong, but I would like to understand.